docs/source/running-autometa.rst

```diff
@@ -2,108 +2,271 @@
+
+.. note::
+    1. Run the following commands directly in ``bash``. Nexflow will automatically submit jobs to SLURM or any other sheduling system.
+    2. You can use `tmux <https://github.com/tmux/tmux/wiki>`_ or `screen <https://www.gnu.org/software/screen/>`_ in case you want to exit the window or disconnect from the server.
+    3. The pipeline must be launched from a node where the ``condor_submit`` command is available, that is, in a common usage scenario, the cluster head node.
+    4. The HTCondor executor for Nextflow_ does not support at this time the HTCondor ability to transfer input/output data to the corresponding job computing node. Therefore the data needs to be made accessible to the computing nodes using a shared file system directory from where the Nextflow_ workflow has to be executed (or specified via the -w option).
+
+With docker (Simplest)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Make sure that you have Docker_ and nextflow_ installed. You can run autometa pipeline using ``nextflow run KwanLab/Autometa -c parameters.config``
+
+In case you want to tweak some of the scripts and modify the pipeline you can clone the repository and then run autometa using ``main.nf``. This is completely optional and would be useful only for development purposes.
+
+.. code-block:: bash
+
+    # Clone the autometa repository
+    git clone git@github.com:KwanLab/Autometa.git
+    # Navigate to the clone repository
+    cd $HOME
+    # Run nextflow
+    nextflow run main.nf
+
+Without docker
+^^^^^^^^^^^^^^
+
+By default autometa's implementation using nextflow_ makes use of Docker_. You can diasble it in either your ``nextflow.config`` file or the ``parameters.config`` file.
+
+Editing ``nextflow.config``:
+
+.. code-block:: bash
+
+    // Find this section of code in nextflow.config
+    docker {
+    enabled = false // <-- change to this
+    // Nextflow will run using the local autometa installation
+    // rather than the docker image specified.
+    fixOwnership = true
+    }
+
+Editing ``parameteres.config``:
+
+.. code-block:: bash
+
+    // Find this section of code in parameteres.config
+    docker.enabled = false // override use of docker to use local Autometa installation
+
+Now install autometa using one of the three install methods specified in :ref:`Install` (Directly using conda or from source). After the install you can run autometa using ``nextflow run KwanLab/Autometa -c parameters.config``. Nextflow_ would use the entrypoints created during the install to run autometa. Make sure to actiavte your conda environment before running incase you have installed using conda.
+
+Other useful options
+^^^^^^^^^^^^^^^^^^^^
+
+``-c`` : In case you have configured nextflow_ with your executor (see below) and have made other modifications on how to run nextflow_ using your ``nexflow.config`` file, you can specify that file using the ``-c`` flag
```

Comment:
This flag is required since you must provide your parameters config file with `-c paramters.config`. Doesn't seem necessarily like an "other" option in this case.